docs/deploy-from-windows.md

```diff
@@ -17,17 +24,25 @@ Wait a minute for Windows to install a few things in the background (it will eve
 5. Restart Windows and then [install Ubuntu from the Windows Store](https://www.microsoft.com/p/ubuntu/9nblggh4msv6).
 6. Run Ubuntu from the Start menu. It will take a few minutes to install. It will have you create a separate user account for the Linux subsystem. Once that's done, you will finally have Ubuntu running somewhat integrated with Windows.
 
+## Install Algo
 
-Install additional packages:
+Run these commands in the Ubuntu Terminal to install a prerequisite package and download the Algo scripts to your home directory. Note that when using WSL you should **not** install Algo in the `/mnt/c` directory due to problems with file permissions.
+
+You may need to follow [these directions](https://devblogs.microsoft.com/commandline/copy-and-paste-arrives-for-linuxwsl-consoles/) in order to paste commands into the Ubuntu Terminal.
 
 ```shell
-sudo apt-get update && sudo apt-get install git build-essential libssl-dev libffi-dev python3-dev python3-pip python3-setuptools python3-virtualenv -y
+cd
+umask 0002
+sudo apt update
+sudo apt install -y python3-virtualenv
+git clone https://github.com/trailofbits/algo
```

Comment:
Is `git` pre installed by default on WSL?

Reply:
I don't believe @TC1977 needed to install `git`.

Reply:
I didn't need to install `git`, and I believe [that's the default for WSL2 at least](https://docs.microsoft.com/en-us/windows/nodejs/setup-on-wsl2#set-up-git-optional).

Reply:
Ok, looks like I can't install WSL2 currently, as I'm not an "Insider" and only have Win10 OS build 18362. Well I reinstalled WSL and Ubuntu 18.04.2 LTS, and `git` still comes pre-installed, as does `nano`.